component/ldap.libsonnet

```diff
@@ -0,0 +1,88 @@
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local oauth = import 'lib/openshift4-oauth.libjsonnet';
+local inv = kap.inventory();
+
+local params = inv.parameters.openshift4_oauth;
+
+local syncConfig(namespace, idp, sa) =
+  local name = 'ldap-sync-' + oauth.RefName(idp.name);
+  local mount = '/etc/sync-config/';
```

Comment:
Would at least include "ldap" in the name. sync-config could be everything, hard to remember and hard to find.

Reply:
Is irrelevant, as it is only inside a container.